src/_scss/components/_pagination.scss

```diff
@@ -1,6 +1,6 @@
 .pagination {
     margin: rem(15);
```

Comment:
You should not have margin declarations in your block level classes:
https://en.bem.info/methodology/quick-start/#block
> The block shouldn't influence its environment, meaning you shouldn't set the external geometry (margin) or positioning for the block.

An alternative would be to either wrap the children inside an element, which has margins, or to have the pagination component's parent component provide the margins.

Reply:
Removed positioning from the block; added margins in the spending explorer table file